deploy/tests/test.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'html/proofer'
+proofer = HTML::Proofer.new("./_site", {
+  :directory_index_file => "index.html",
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.